views/pages/details.html

```diff
@@ -43,6 +43,12 @@
   {{#record.status.courtOrderRevoked}}
     <div class="flag">{{#t}}flags.courtOrderRevoked{{/t}}</div>
   {{/record.status.courtOrderRevoked}}
+
+  {{#record.previousRegistration.systemNumber}}
+    <div class="flag warning">{{#t}}flags.previousRegistration{{/t}}
```

Comment:
I don't think this is a warning necessarily. Let's have it as a normal flag for now and let the business decide what they want.